p2p/protocol/circuitv2/relay/metrics.go

```diff
@@ -0,0 +1,262 @@
+package relay
+
+import (
+	"time"
+
+	"github.com/libp2p/go-libp2p/p2p/metricshelper"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const metricNamespace = "libp2p_relaysvc"
+
+var (
+	status = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Namespace: metricNamespace,
+			Name:      "status",
+			Help:      "Relay Current Status",
+		},
+	)
+
+	reservationTotal = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: metricNamespace,
+			Name:      "reservation_total",
```

Comment:
I think the naming convention says this should be plural.
```suggestion
			Name:      "reservations_total",
```